packages/opentelemetry-bundler-utils/src/get-instrumentation-config.ts

```diff
@@ -41,11 +42,7 @@ function configGenerator<T extends { enabled?: boolean }>(
   config?: T
 ): string | undefined {
   if (!config) return;
-  return JSON.stringify(
-    Object.fromEntries(
-      Object.entries(config).filter(([, v]) => typeof v !== "function")
-    )
-  );
+  return stringify(config, null, 2);
```

Comment:
nit: Users bundling their code are likely to not read the produced code (and possibly minifying), so do we need to pretty print?
```suggestion
  return stringify(config);
```